src/impl_methods.rs

```diff
@@ -930,7 +930,7 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
     ///          [6, 6, 7, 7, 8, 8, 0],
     ///          [6, 6, 7, 7, 8, 8, 0]]));
     /// ```
-    pub fn exact_chunks_mut<E>(&mut self, chunk_size: E) -> ExactChunksMut<A, D> 
+    pub fn exact_chunks_mut<E>(&mut self, chunk_size: E) -> ExactChunksMut<A, D>
```

Comment:
My vim settings remove trailing whitespaces on save... Got to change it when I am working on ndarray :P

Reply:
I have Emacs set up the same way. Fwiw, I've found it useful to have the "save" keybinding set to remove trailing whitespace, but preserve the ability to call the `save-buffer` command directly if desired to save without removing whitespace.

Anyway, I've submitted PR #464 to remove all the trailing whitespace in `ndarray` so that we won't have to worry about this issue anymore.

For the purpose of this PR, though, @bluss is right that the PR should not reformat lines unrelated to the changes.

Reply:
Yes, I was thinking about doing something similar to that in my `.init.vim`.
Until #464 lands I guess I'll be restoring whitespaces once the code has been approved and it is just waiting to be merged - final clean-up, so to speak.